shared/js/background/events.es6.js

```diff
@@ -287,7 +287,7 @@ chrome.omnibox.onInputEntered.addListener(function (text) {
         active: true
     }, function (tabs) {
         chrome.tabs.update(tabs[0].id, {
-            url: 'https://duckduckgo.com/?q=' + encodeURIComponent(text) + '&bext=' + localStorage.os + 'cl'
+            url: 'https://duckduckgo.com/?q=' + encodeURIComponent(text) + '&bext=' + utils.getOsName() + 'cl'
         })
     })
```

Comment:
This whole feature is not working. I'm not sure why we have it in the first place. To test this you have to type "ddg" in the omnibox, click TAB, and then search for something. It won't work and it didn't work before this PR. Let's remove the whole thing (it's this one listener and note in manifest.json)?

Reply:
Hm, wait, it's working on production, something weird is going on because that `tabs[0].id` definitely shouldn't work.

Reply:
Yeah, it thows errors but it works somehow. `update` defaults to current tab and this first param is optional, so lets remove it.

Reply:
This works for me (in Chrome). Perhaps it's chrome only?

Reply:
Yeah, my bad. Bug only exists when debugging